app/frontend/src/components/commons/Modal/index.css.ts

```diff
@@ -0,0 +1,48 @@
+import { style } from '@vanilla-extract/css';
+
+export const background = style({
+  position: 'absolute',
+  top: 0,
+  left: 0,
+  width: '100%',
+  height: '100%',
+  background: 'rgba(14, 31, 24, 0.05)',
+  backdropFilter: 'blur(0.1rem)',
+});
+
+export const buttonArea = style({
+  display: 'flex',
+  justifyContent: 'stretch',
+  gap: '1.6rem',
+});
+
+export const container = style({
+  display: 'flex',
+  flexDirection: 'column',
+  position: 'fixed',
+  top: '50%',
+  left: '50%',
+  minHeight: '20rem',
+  minWidth: '32rem',
+  padding: '1.6rem',
+  borderRadius: '0.8rem',
+  boxShadow: '0 0.4rem 0.8rem rgba(0, 0, 0, 0.25)',
+  transform: 'translate(-50%, -50%)',
+});
```

Comment:
```suggestion
export const container = style({
  flexDirection: 'column',
  position: 'fixed',
  top: '50%',
  left: '50%',
  minHeight: '20rem',
  minWidth: '32rem',
  padding: '1.6rem',
  borderRadius: '0.8rem',
  boxShadow: '0 0.4rem 0.8rem rgba(0, 0, 0, 1)',
  transform: 'translate(-50%, -50%)',

  selectors: {
    [`&[open]`]: {
      display: 'flex',
    },
  },
});
```

이런 식으로 작성하면 open 상태에 따라 css를 작성하기 때문에 dialog에 open 속성이 빠지면 (dialog.close() 메소드를 호출하면) 모달이 사라져요

Reply:
해당 내용 적용하였습니다! 엄청난 기능이네요... 감사합니다🥹

Reply:
저도 최근에 알게 된 기능인데 html 활용할 수 있는 태그가 정말 많다는 걸 다시 한번 느꼈습니다